packages/teleport-project-generator/src/file-handlers.ts

```diff
@@ -119,18 +127,31 @@ const createHTMLEntryFileChunks = (uidl: ProjectUIDL, options: EntryFileOptions)
     addChildNode(headNode, titleTag)
   }
 
-  // For frameworks that need to inject and point out the generated build files
+  /* For frameworks that need to inject and point out the generated build files
+  or adding some script tags in head or body */
   if (customScriptTags.length > 0) {
     customScriptTags.forEach((tag: CustomScriptTag) => {
-      const { type, path } = tag
+      const { type, path, content } = tag
+      const { target } = tag || { target: 'head' }
+
+      const targetNode = target === 'body' ? bodyNode : headNode
       const scriptTag = createHTMLNode('script')
-      if (type === 'module') {
-        addAttributeToNode(scriptTag, 'type', type)
-      } else {
-        addAttributeToNode(scriptTag, 'nomodule', '')
+
+      if (type) {
+        type === 'module'
```

Comment:
can you please explain the logic here in a comment?

Reply:
This needs a small refactor, the current logic won't work for all edge cases. I will update this now 👍